## EvolutionaryRate_RegulatoryGenome

The development of explanatory models of protein sequence evolution has broad implications for our understanding of cellular biology, population history, and disease etiology. Here we analyze the GTEx transcriptome resource to quantify the effect of the transcriptome on protein sequence evolution in a multi-tissue framework. We find substantial variation among the central nervous system tissues in the effect of expression variance on evolutionary rate, with highly variable genes in the cortex showing significantly greater purifying selection than highly variable genes in subcortical regions (Mann-Whitney U p=1.4x10<sup>-4</sup>). The remaining tissues cluster in observed expression correlation with evolutionary rate, enabling evolutionary analysis of genes in diverse physiological systems, including digestive, reproductive, and immune systems. Importantly, the tissue in which a gene attains its maximum expression variance significantly varies (p=5.55x10<sup>-284</sup>) with evolutionary rate, suggesting a tissue-anchored model of protein sequence evolution. Using a large-scale reference resource, we show that the tissue-anchored model provides a transcriptome-based approach to predicting the primary affected tissue of developmental disorders. Using gradient boosted regression trees to model evolutionary rate under a range of model parameters, selected features explain up to 62% of the variation in evolutionary rate and provide additional support for the tissue model. Finally, we investigate several methodological implications, including the importance of evolutionary-rate-aware gene expression imputation models using genetic data for improved search for disease-associated genes in transcriptome-wide association studies. Collectively, this study presents a comprehensive transcriptome-based analysis of a range of factors that may constrain molecular evolution and proposes a novel framework for the study of gene function and disease mechanism.

## References

The regulatory genome constrains protein sequence evolution: Implications for the search for disease-associated genes

Patrick Evans<sup>1</sup>, Nancy J. Cox<sup>1</sup>, Eric R. Gamazon<sup>1,2,3,4</sup>

<sup>1</sup>Division of Genetic Medicine, Vanderbilt University Medical Center, Nashville, TN 37232  
<sup>2</sup>Data Science Institute, Vanderbilt University, Nashville, TN 37232  
<sup>3</sup>Clare Hall, University of Cambridge, Cambridge, CB3 9AL, United Kingdom  
<sup>4</sup>MRC Epidemiology Unit, University of Cambridge, Cambridge, CB2 0QQ, United Kingdom  

Questions:  

Patrick Evans, Ph.D. <patrick.evans@vanderbilt.edu>  
Eric R. Gamazon, Ph.D. <ericgamazon@gmail.com>  



